pkg/skaffold/deploy/kpt/kpt_test.go

```diff
@@ -1087,22 +1087,58 @@ func TestVersionCheck(t *testing.T) {
 				AndRunOut("kustomize version", `{Version:v0.0.1 GitCommit:a0072a2cf92bf5399565e84c621e1e7c5c1f1094 BuildDate:2020-06-15T20:19:07Z GoOs:darwin GoArch:amd64}`),
 			kustomizations: map[string]string{"Kustomization": `resources:
 				- foo.yaml`},
-			shouldErr: true,
-			error: fmt.Errorf("you are using kustomize \"v0.0.1\"\n"+
-				"Please update your kustomize version to >= %v\n"+
-				"See kustomize installation: %v", kustomizeMinVersion, kustomizeDownloadLink),
+			shouldErr: false,
+			error:     nil,
```

Comment:
can you find a way to check if the warn message is correct?

Reply:
The only way to do that without changing the function is to catch the stdout. I've implemented test utils for this locally and they work well. A `SetStdout()` function to catch the stdout in a temporary file and a set of `CheckLog*()` functions that mimic the `CheckError*()` functionalities.

These are obviously best placed in the 'testutil/util.go' but seeing your previous reservations about code changes I'm guessing that is not a desirable way to go. I could move them into `kpt_test.go` file but that will make things more messy imo.

What do you advise?